docs/src/Callout.doc.js

```diff
@@ -22,168 +47,354 @@ card(
-          '- onClick: Callback fired when the button component is clicked (pressed and released) with a mouse or keyboard.',
-          `- onNavigationOptions: onNavigationOptions works in conjunction with a Provider. Pass custom props to onNavigation. See Provider for examples. onNavigation's type is flexible. Each key's value is a React.Node or an event handler function. Optional with href.`,
-          'Accessibility: `accessibilityLabel` populates aria-label. Screen readers read the `accessibilityLabel` prop, if present, instead of the button `text`.',
-        ],
+        description: `
+          Secondary action for people to take on the Callout
+
+          href: If \`href\` is supplied, the action will serve as a link. If no \`href\` is supplied, the action will be a button.
+
+          The \`accessibilityLabel\` should follow the [Accessibility guidelines](#Accessibility).
+        `,
         href: '',
       },
       {
         name: 'type',
         type: `"error" | "info" | "warning"`,
         required: true,
         defaultValue: null,
-        description: [
-          'Select a type of callout:',
-          '-`error`: To provide error messaging to the user.',
-          '-`info`: To provide general information to the user.',
-          '-`warning`: To provide a warn message to the user.',
-        ],
-        href: '',
+        description: `
+        The type of Callout. See the [Variants](#Variants) to learn more
+        `,
       },
       {
         name: 'title',
         type: 'string',
-        required: false,
         defaultValue: null,
-        description: [
-          'Heading to render inside the callout above the message to convey the callout topic to the user. The message text has a fixed size.',
-        ],
-        href: '',
+        description: `Brief title summarizing the Callout. Content should be localized.`,
       },
     ]}
   />,
 );
 
 card(
-  <Example
-    name="Info Example"
-    defaultCode={`
+  <MainSection name="Best practices">
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="lg"
+        type="do"
+        description="Use Callouts for messages coming from the product or user interaction."
+        defaultCode={`
+          <Callout
+            type="error"
+            title="Pinterest tag needs attention"
+            iconAccessibilityLabel="Error"
+            message="Your tag has errors, so information may be outdated. Fix your tag for the most accurate metrics."
+            primaryAction={{href: "https://pinterest.com", label:"Fix tag"}}
+
+          />
+        `}
+      />
+      <MainSection.Card
+        cardSize="lg"
+        type="do"
```

Comment:
I know it's different given the wider content, but it seems odd to me to not alternate Do's and Don't's

Reply:
Chatted with @anniemochi about this and I think the guideline is to stack Dos then Don'ts if they're large. Annie what do you think?

Reply:
Ah ok, that's fine with me if that's our standard. 👍